controllers/acs_device_info.js

```diff
@@ -2355,6 +2377,29 @@ const syncDeviceData = async function(
       }
     }
 
+    if (data.wifi2.hidden && data.wifi2.hidden.value !== 'undefined') {
+      let hidden = utilHandlers.convertToBoolean(data.wifi2.hidden.value);
+      if (device.has_synced_wifi_hidden === undefined) {
```

Comment:
Melhor verificar se é `false` ou `undefined`, não? Pega o legado e qualquer caso em que o sync está falso
```suggestion
      if (!device.has_synced_wifi_hidden) {
```